notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -0,0 +1,687 @@
+    "3. Combine the aligned images using `AstroDrizzle`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "from astropy.io import ascii\n",
+    "from astropy.io import fits\n",
+    "from astropy.table import Table\n",
+    "from astropy.visualization import ZScaleInterval\n",
+    "from astroquery.mast import Observations\n",
+    "from ccdproc import ImageFileCollection\n",
+    "from drizzlepac import tweakreg\n",
+    "from drizzlepac import astrodrizzle\n",
+    "from IPython.display import Image\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Download the Data \n",
+    "\n",
+    "In this example, we align three F606W full-frame 339 second ACS/WFC images of the globular cluster NGC 104 from ACS/CAL Program [10737](http://www.stsci.edu/cgi-bin/get-proposal-info?id=10737&observatory=HST).  These observations were acquired over a 3-month period in separate visits and at different orientations. We will use calibrated, CTE-corrected `*_flc.fits` files, which are available for both WFC3/UVIS and ACS/WFC detectors. \n",
+    "\n",
+    "            j9irw3fwq_flc.fits\n",
+    "            j9irw4b1q_flc.fits\n",
+    "            j9irw5kaq_flc.fits\n",
+    "            \n",
+    "First, we use `astroquery` to look for the desired datasets and retrieve them from MAST. \n",
+    "\n",
+    "You may query on a large number of parameters, but to obtain these specific datasets we will only need to pass in a few: `obstype = 'all'` to include both calibration and GO programs in the search, `obs_collection = 'HST'` to exclude Hubble Legacy Archive images, and finally `obs_id` to search for a list of the dataset IDs. We finally select only 'FLC' data products using the parameter `productSubGroupDescription` while downloading the files. Because the second 'FLC' file for visit W4 is part of a dithered association, the `obs_id` for that assocation will need to be used instead of the individual filename.  \n",
+    "\n",
+    "**Note**: The next cell may take awhile to complete or may need to be run more than once, as errors may occasionally arise from a failed server connection."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Query the data\n",
+    "obsTable = Observations.query_criteria(obstype='all', \n",
+    "    obs_collection='HST',\n",
+    "    obs_id=['j9irw3fwq', 'j9irw4040', 'j9irw5kaq'])\n",
+    "\n",
+    "# Download the files\n",
+    "products = Observations.get_product_list(obsTable)\n",
+    "Observations.download_products(products,download_dir='', mrp_only=False, productSubGroupDescription='FLC')\n",
```

Comment:
Maybe there should be linebreaks in these function arguments?  When looking at the notebook rendered it's rather hard to read, and seems a bit odd given that the cell above seems to not follow that.